tableau-databricks/manifest.xml

```diff
@@ -152,7 +152,9 @@ limitations under the License.
         <customization name='CAP_SUPPRESS_ENUMERATE_TABLES_VIA_SQL' value='yes' />
         <customization name='CAP_SUPPRESS_ENUMERATE_SCHEMAS_VIA_SQL' value='yes' />
         <customization name='CAP_ODBC_SUPPRESS_ENUMERATE_SCHEMA_WITHOUT_CATALOG' value='yes' />
-	<customization name='CAP_ODBC_SUPPRESS_INFO_SCHEMA_SCHEMAS' value='yes' />
+        <customization name='CAP_ODBC_SUPPRESS_INFO_SCHEMA_SCHEMAS' value='yes' />
+        <!--enable cross-catalog join within same connection-->
+        <customization name=' CAP_ODBC_INCLUDE_CATALOG_NAME' value='yes' />
```

Comment:
Is this a new capability exposed by the SDK? Before you'd have to create two different connections and then set the catalog to join with via the `Catalog` ODBC conf in the connection string. Did we confirm with Tableau folks this works as expected?